src/main/java/teammates/common/util/Const.java

```diff
@@ -521,6 +521,7 @@
         public static final String COMMENTS = "comments";
         public static final String TEAMMATES = "teammates";
     
+        public static final String STATUS_MESSAGES_LIST = "statusMessagesToUser";
```

Comment:
Are the constants STATUS_MESSAGE and STATUS_MESSAGE_COLOR still used? Look through and remove them from the code if they are not.